CHANGELOG.md

```diff
@@ -1,22 +1,50 @@
-1.9.0 (in progress)
+1.10.0 (in progress)
 ===================
 
 New Features
 ------------
 
 - A RemoteParticipant now maintains a collection of RemoteTrackPublications in a
-  new `.trackPublications` property. It also maintains media specific
+  new `.trackPublications` property. It also maintains kind-specific
   RemoteTrackPublication collections (`.audioTrackPublications`,
-  `.dataTrackPublications` and `.videoTrackPublications`). A RemoteTrackPublication
-  represents a Track that was published to the Room by a RemoteParticipant. A
-  "trackPublished" event is emitted on the RemoteParticipant (and subsequently on
-  the Room) whenever a Track is published. A "trackUnpublished" event is emitted
-  on the RemoteParticipant whenever a Track is unpublished.
+  `.dataTrackPublications` and `.videoTrackPublications`). A "trackPublished"
+  event is emitted on the RemoteParticipant (and subsequently on the Room) whenever
+  a Track is published. A "trackUnpublished" event is emitted on the RemoteParticipant
+  whenever a Track is unpublished.
   
   ```js
   participant.on('trackPublished', publication => {
     console.log('A new Track was published!', publication);
+  });
+
+  remoteParticipant.on('trackUnpublished', publication => {
+    console.log('A new Track was unpublished!', publication);
+  });
+  ```
+
+RemoteTrackPublication Guide
+----------------------------
+
+- A RemoteTrackPublication represents a Track that was published to the Room by
+  a RemoteParticipant.
+
+- A RemoteTrackPublication has the following properties:
+  * `isSubscribed` - Whether the LocalParticipant has subscribed to the Track
```

Comment:
Nitpick: insert newline here if we keep the list; however, I don't necessarily think we need the list. The code sample looks sufficient to me.